views/index.hbs

```diff
@@ -0,0 +1,33 @@
+<main>
+  <div class='container my-5'>
+    <div class='row p-4 pb-0 pe-lg-0 pt-lg-5 align-items-center rounded-3 border shadow-lg'>
+      <div class='col-lg-7 p-3 p-lg-5 pt-lg-3'>
+        {{#if darkmode}}
```

Comment:
not sure how feasible it is, but with bootstrap you can control the "theme-colors" – basic idea here would be to add a style tag to the layout that based on the darkmode setting sets the css variables on the :root object (or wherever bootstrap defines them) to their dark/light variants. this way you could have a lot less code duplication.

Reply:
i.e. instead of changing the button colors on each button/element separately, just change the underlying variable for it :)

Reply:
see something like this: 
https://stackoverflow.com/questions/68667437/create-new-color-scheme-for-dark-light-mode-in-bootstrap-sass/68670276#68670276

Reply:
there's also this:
https://github.com/vinorodrigues/bootstrap-dark-5